helm/charts/hydra/templates/deployment.yaml

```diff
@@ -157,6 +157,10 @@ spec:
             {{- toYaml .Values.deployment.securityContext | nindent 12 }}
           lifecycle:
             {{- toYaml .Values.deployment.lifecycle | nindent 12 }}
+      {{- if .Values.deployment.extraInitContainers }}
+      initContainers:
+{{ tpl .Values.deployment.extraInitContainers . | indent 8 }}
```

Comment:
```suggestion
{{ toYaml ( tpl .Values.deployment.extraInitContainers . ) | indent 8 }}
```

Reply:
We learned in https://github.com/ory/k8s/pull/310 that this won't work